_includes/components/svg.html

```diff
@@ -5,8 +5,10 @@
 - width {Number} requires css units
 {% endcomment %}
 
-{% assign iconPath = "svg/" | append: include.filename | append: ".svg" %}
+{% if include.filename and include.filename != "" %}
```

Comment:
Liquid FEELS a little like Twig, but without a lot of feature... I needed a way to not include the icon from a button if not specified. I wan't sure about setting as `false` since it's TECHNICALLY supposed to be a string. Still getting the hang of using Liquid/Jekyll to do components...